docs/guides/proofreading.md

```diff
@@ -30,49 +53,51 @@ Once SLEAP has measured the similarity between all the candidates and the instan
 
 Finally, you have an option second-pass method which “cleans” the resulting identities. To use this method, you specify a target number of tracks (i.e., how many animals there are in your video). SLEAP then goes frame by frame and removes instances over this target number.
 
-Once you have the desired number of instances in every frame, SLEAP connects identities with a simple heuristic: if exactly one track identity was dropped from frame N and exactly one new track identity was added in frame N+1, it matches up the dropped and the new tracks.
+To set the number of tracks, navigate to the Inference Pipeline via Predict >>
+Inference, then:
 
+1) Specify the **Tracker (cross-frame identity) Method**
+2) Uncheck the **No target** checkbox
+3) Specify the **Target Number of Instances Per Frame**
+4) Check the **Cull to Target Instance Count** checkbox
```

Comment:
Finally, you have an optional second-pass method which “cleans” the resulting identities with "**Cull to Target Instance Count**". To use this method, you specify a target number of tracks by setting the "**Target Number of Instances Per Frame**" (i.e., how many animals there are in your video). SLEAP then goes frame by frame and removes (or culls) instances over this target number. To cull to a target number of instances per frame, navigate to the Inference Pipeline via Predict >> Inference, then:
1) Specify the **Tracker (cross-frame identity) Method**
2) Uncheck the **No target** checkbox
3) Specify the **Target Number of Instances Per Frame**
4) Check the **Cull to Target Instance Count** checkbox